CHANGES.txt

```diff
@@ -1,4 +1,5 @@
 ﻿Current
+Fixed: GITHUB-1334: Priority order broken when combined with dependencies that have failures (Krishnan Mahadevan)
```

Comment:
Don't forget to add "Close #xxx" in the commit message. It will close the issue automatically.